meetings/2020-03/april-1.md

```diff
@@ -0,0 +1,1028 @@
+
+BN: It feels like there's a lot in this proposal right now. Some people will say that syntax is expensive. I wonder if the runtime part (i.e. the record and tuple functions, which I think are totally adequate for getting the sort of immutable triple equals objects you want) . Might be something that we can prioritize as a proposal in itself. Then a follow-up proposal to add the shorthand hash syntax, or do we need to include everything that's eventually going to be in the proposal up front?
+
+RBU: From the champions group perspective, I think there’s always  a fine line you have to draw between how fine-grained and how large you want these proposals to be. We're trying to strike that balance by splitting out RefCollections, removing with syntax, etc. I still think the syntax is very valuable for two reasons: 1) you don't just repeat the word Record or Tuple over and over again. 2) We already have significant runtime experience with this, with libraries that implement immutable data structures in userland. So the goal with record and tuple is to not only provide an alternative to those, but to provide something with a very terse syntax integrated into the language so you use it as much as possible. The addition of the shorthand syntax adds a lot of value to the proposal.
+
+BN: I agree and I like this syntax, but it’s important to mention that it’s completely separable.
+
+RBU: Yeah, and like I said the fine line to draw between where to cut for future proposals is complicated. I’m still of the opinion that it’s worth including with this proposal but am interested in hearing what the committee thinks about this.
+
+MLS: I’m concerned that the update on this proposal came with two more stage 0 proposals, and if they’re needed to make these proposals useful, then I think those features should be considered as part of this proposal.
+
+RBU: Even though we're not presenting on them this meeting we intend to iterate on them in parallel. The proposals being split gives us the advantage of iterating more quickly on the things that people agree on, like the core semantics of record & tuple. I agree that RefCollection and deep path properties are important for this proposal, but they're intended to solve sharp edges. I think Record and Tuple are still very valuable without them.
+
+RRD: Yes, mostly RefCollection is something we don’t think we need, it’s something to address concerns. Honestly we don’t think both of those proposals are essential, though we do like them.
+
+JRL: I just wanted to voice strong support for RefCollection. The ability to place holes in immutable trees is very useful. There is high value in holding references to mutable data out-of-band relative to the immutable structure. (excellent note taking!)
+
+Chair: Champions group, please ensure that the remaining comments about RefCollection are discussed.
```

Comment:
@rickbutton and @rricard -- does anything need to be captured here?

Reply:
We are following up in the RefCollection issue tracker already. Thanks!